packages/design-tokens/src/css/typography.css

```diff
@@ -101,11 +92,11 @@
   --typography-s-body-xs-medium-font-size: var(--font-size-2);
   --typography-s-body-xs-medium-line-height: var(--line-height-2);
   --typography-s-body-xs-medium-font-weight: var(--font-weight-medium);
-  --typography-s-body-xs-medium-letter-spacing: var(--letter-spacing-0);
+  --typography-s-body-xs-medium-letter-spacing: var(--letter-spacing-1);
   --typography-s-body-xs-font-size: var(--font-size-2);
   --typography-s-body-xs-line-height: var(--line-height-2);
   --typography-s-body-xs-font-weight: var(--font-weight-regular);
-  --typography-s-body-xs-letter-spacing: var(--letter-spacing-0);
+  --typography-s-body-xs-letter-spacing: var(--letter-spacing-1);
```

Comment:
Revert these changes as they should be in https://github.com/MetaMask/metamask-design-system/pull/697